### PR TITLE
Adding week 8 blog post

### PR DESCRIPTION
**Pull request by xtanion**
Blog is visible in `blogs.html`

### CODE REVIEW

File: docs/source/posts/2022/2022-08-09-week-08-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+This week I had to fix the transformation issue in glTF animation. As discussed in the last meeting, I disabled vertex transformation and applied the transformation after the actor was formed.
+
+- Creating transformation matrix as it is.
```
Comment (ganimtron-10) [inline docs/source/posts/2022/2022-08-09-week-08-shivam.rst:14]: Can you explain what this line means?
I am not getting what `as it is` means.

**Review by ganimtron-10 — COMMENTED** [review]
Hello @xtanion ,
Your blog looks nice!! 
PTAL at the below comments.
Thanks!

**Comment by skoudoro** [comment]
waiting for @m-agour review to start here

File: docs/source/posts/2022/2022-08-09-week-08-shivam.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+This week I had to fix the transformation issue in glTF animation. As discussed in the last meeting, I disabled vertex transformation and applied the transformation after the actor was formed.
+
+- Creating transformation matrix using the TRS data from nodes.
+- Extract Translation, Rotation, and Scale matrice from the transformation matrix.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-09-week-08-shivam.rst:15]: Typo in `matrices`

**Review by m-agour — COMMENTED** [review]
Hello @xtanion,
I only found the following typo. After that, it's good to go.
Also sorry for late review.

**Review by skoudoro — APPROVED** [review]
LGTM! thanks @xtanion, merging!

**Pull request merged**